includes/admin/helper/class-wc-helper.php

```diff
@@ -1046,12 +1051,23 @@ public static function get_local_woo_plugins() {
 			$data['_product_id']      = absint( $product_id );
 			$data['_file_id']         = $file_id;
 			$data['_type']            = 'plugin';
+			$data['slug']             = dirname( $filename );
 			$woo_plugins[ $filename ] = $data;
 		}
 
+		set_transient( 'wc_helper_installed_woo_plugins', $woo_plugins, DAY_IN_SECONDS );
```

Comment:
Just a thought; store this transient as long as possible, and flush when a plugin is activated.

https://codex.wordpress.org/Plugin_API/Action_Reference/activated_plugin

This will mean cache is used more frequently.

Reply:
This code's gone now, using `active_plugins` instead, which is more conventional, and means we don't need to make changes in helper. 8dbda1b9c962a93be7d17ffa8ac55c2639f641e1